Can't install on windows

Hey, when I run cargo install on windows I'm having issues with termion

```
   Compiling termion v1.5.5
error[E0433]: failed to resolve: maybe a missing crate `sys`?
  --> C:\Users\xxxx\.cargo\registry\src\github.com-1ecc6299db9ec823\termion-1.5.5\src\lib.rs:24:9
   |
24 | pub use sys::size::terminal_size;
   |         ^^^ maybe a missing crate `sys`?
```

I haven't tried to install it on linux yet, is this something that may be exclusive to my computer or something?

At the end it says it has failed to compile dijo v0.1.3

It looks like ncurses isn't supported on Windows:

```
error: failed to run custom build command for `ncurses v5.99.0`
...
c1: fatal error C1083: Cannot open source file: 'D:\apps\dijo\target\release\build\ncurses-a01b84948c9e0b70\out\chtype_size.c': Permission denied
```
Maybe consider using [pancurses](https://github.com/ihalila/pancurses)?

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -82,3 +82,38 @@ jobs:
           asset_name: dijo-x86_64-apple
           tag: ${{ github.ref }}
           overwrite: true
+
+  build-windows:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cache Rust dependencies
+        uses: actions/cache@v1.0.1
+        with:
+          path: target
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-build-
+      - name: Install latest rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: beta
+          target: x86_64-pc-windows-msvc
+          default: true
+          override: true
+
+      - name: Build for windows
+        run: |
+          cargo build --all --release
+          strip target/release/dijo
+
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: target/release/dijo
+          asset_name: dijo-x86_64-windows
+          tag: ${{ github.ref }}
+          overwrite: true

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ notify = "4.0"
 [dependencies.cursive]
 version = "0.15"
 default-features = false
-features = ["termion-backend"]
+features = ["crossterm-backend"]
 
 [dependencies.chrono]
 version = "0.4"

--- a/src/main.rs
+++ b/src/main.rs
@@ -12,7 +12,7 @@ use crate::command::{open_command_window, Command};
 use crate::utils::{load_configuration_file, AppConfig};
 
 use clap::{App as ClapApp, Arg};
-use cursive::termion;
+use cursive::crossterm;
 use cursive::views::{LinearLayout, NamedView};
 use lazy_static::lazy_static;
 
@@ -62,7 +62,7 @@ fn main() {
             println!("{}", h);
         }
     } else {
-        let mut s = termion().unwrap();
+        let mut s = crossterm().unwrap();
         let app = App::load_state();
         let layout = NamedView::new(
             "Frame",

